README.md

```diff
@@ -7,7 +7,7 @@ Compatibility
 ------------------------------------------------------------------------------
 
 * Ember.js v3.16 or above
-* Ember CLI v2.13 or above
+* Ember CLI v3.16 or above
```

Comment:
The ember cli version is supposed to be in sync with the Ember.js version, but it took a long time before they actually updated it :smile:.

more info: https://github.com/ember-cli/ember-cli/pull/9491